dom.bs

```diff
@@ -499,6 +505,15 @@ list of tuples, each of which consists of an <b>item</b> (an {{EventTarget}} obj
 object). A tuple is formatted as (<b>item</b>, <b>target</b>, <b>relatedTarget</b>). A <a
 for=Event>path</a> is initially the empty list.</p>
 
+For web compatibility, the <a>current global object</a> has an
+<dfn attribute for=Window><code>event</code></dfn> attribute that returns the
+<dfn export id=concept-current-event>current event</dfn>, which is initially
+set to undefined.
```

Comment:
You also need to define what current event is associated with. Also, we should probably test that this quirk is not present in workers? And ensure the algorithms work in a worker context as well, since events don't just dispatch inside a window.